### PR TITLE
Feature email server

### PR DESCRIPTION
**Pull request by mcawful**

### CODE REVIEW

File: SurveyService/src/main/java/com/revature/SandBox.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.revature;
```
Comment (jurczakn) [inline SurveyService/src/main/java/com/revature/SandBox.java]: remove

**Review by jurczakn — CHANGES_REQUESTED** [review]
clean up pr, all ignored files need to be removed

File: SurveyService/src/main/java/com/revature/SandBox.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import com.revature.response.EmailResponse;
+
+public class SandBox {
+	public static void main(String[] args) throws JsonProcessingException {
+		final Set<String> malformedEmails = Sets.newHashSet();
+		final Set<String> tokenFailedEmails = Sets.newHashSet();
+		final Set<String> sendFailedEmails = Sets.newHashSet("acacia.holliday@revature.net");
+		final String statusMessage = "";
+		EmailResponse response = new EmailResponse(malformedEmails, tokenFailedEmails, sendFailedEmails, statusMessage);
+		//EmailResponse response = null;
```
Comment (jurczakn) [inline SurveyService/src/main/java/com/revature/SandBox.java]: remove commented out code

**Review by jurczakn — CHANGES_REQUESTED** [review]
.p12 file needs to be removed
remove commented out code

**Comment by chrisbreniser** [comment]
Fixed requests:
- removed .p12 file. 
- deleted sandbox.java
- removed commented out code

**Review by jurczakn — APPROVED** [review]

**Pull request merged**